### ISSUE TITLE
Missing aria labels on the network catalog page

### ISSUE THREAD

**Issue by tw77** [pressbooks/pressbooks-network-catalog#500]
A WAVE report for any network catalog page reveals missing / empty / orphaned form labels. Missing aria labels were previously fixed within https://pressbooks.directory/; this still needs to be done for the network catalog feature.

**Comment by SteelWagstaff** [comment]
@tw77 if you can provide details of which labels are missing (i.e. the WAVE report) and a reference to where/how they were fixed in the Directory, I'd be happy to open a PR to address this week.

**Comment by arzola** [comment]
https://github.com/pressbooks/pressbooks-network-catalog/pull/511

**Comment by tw77** [comment]
In dev network catalogs, the PR resolves the missing form label and empty form label errors that WAVE flags in production network catalogs. Ready for release.
